server.js

```diff
@@ -1639,6 +1639,15 @@ cache(function(data, match, sendBadge, request) {
             versions.push(matched[1]);
           }
         }
+        ['2', '3'].forEach(function(version) {
+            if (versions.some(function(element, index, array) { return new RegExp('^' + version + '\\.\\d$').exec(element); })) {
+                versions.forEach(function(element, index, array) {
+                    if (element === version) {
+                        array.splice(index, 1);
```

Comment:
I'm not a fan of looping over something that changes under our feet. Could we filter `versions` and assign the new list to `versions` afterwards?